sandworm/lib/web.ml

```diff
@@ -0,0 +1,63 @@
+open Tyxml.Html
+
+let bucket_url path =
+  Format.sprintf "https://dune-binary-distribution.s3.fr-par.scw.cloud/%s" path
+;;
+
+let title = title (txt "Dune Dev Preview")
+let css = link ~rel:[ `Stylesheet ] ~href:"main.css" ()
+
+let plausible_js =
+  script
+    ~a:
+      [ Tyxml.Html.Unsafe.string_attrib "data-domain" "dune.ci.dev"
+      ; a_defer ()
+      ; a_src (Xml.uri_of_string "https://plausible.ci.dev/js/script.file-downloads.js")
+      ]
+    (txt "")
+;;
+
+let headers = [ css ]
+let main_title = h2 [ txt "Dune distribution binary" ]
+
+let install =
+  [ p
+      [ txt
+          "After downloaded and extracted, you could use the command below to install it."
```

Comment:
The wordking needs to be adapted. I guess it is actually generated here.

Reply:
It's not generated, but it definitely needs to be reworded. It acted more as a placeholder.